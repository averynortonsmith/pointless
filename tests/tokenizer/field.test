.foo
4.bar