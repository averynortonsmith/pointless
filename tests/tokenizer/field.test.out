  1:1  [ Field        ] .foo
  1:5  [ Newline      ] 
  2:1  [ Number       ] 4
  2:2  [ Field        ] .bar
  2:6  [ EOF          ] 
